src/net_processing.cpp

```diff
@@ -148,6 +148,14 @@ namespace {
     std::unique_ptr<CRollingBloomFilter> recentRejects GUARDED_BY(cs_main);
     uint256 hashRecentRejectsChainTip GUARDED_BY(cs_main);
 
+    /**
+     * Filter for transactions that have been recently confirmed.
+     * We use this to avoid requesting transactions that have already been
+     * confirnmed.
+     */
+    RecursiveMutex cs_recent_confirmed_transactions;
+    std::unique_ptr<CRollingBloomFilter> recent_confirmed_transactions GUARDED_BY(cs_recent_confirmed_transactions);
```

Comment:
nit: Globals should start with `g_` otherwise it might incorrectly look like they live on the stack (as opposed to the heap)